caffe2/distributed/file_store_handler.cc

```diff
@@ -122,6 +122,11 @@ int64_t FileStoreHandler::add(
   return 0;
 }
 
+int64_t FileStoreHandler::getNumKeys() {
+  CHECK(false) << "getNumKeys not implemented for FileStoreHandler";
```

Comment:
shouldn't this be `TORCH_CHECK`?

Reply:
This is in the Caffe2 codebase so I don't think we should use `TORCH_CHECK`. We could use `CAFFE_ENFORCE`, but the other unimplemented function in this class used `CHECK(false)` so I used it as well.